src/main/java/com/gizasystems/automationexercise/pages/HomePage.java

```diff
@@ -8,24 +8,28 @@ public class HomePage {
     // Variables
     private SHAFT.GUI.WebDriver driver;
     private String url = System.getProperty("baseUrl");
-
     // Locators
     private final By featuredItems_div = By.cssSelector("div.features_items");
     private final By recommendedItems_div = By.cssSelector("div.recommended_items");
     private final By categoryTitle_div= By.xpath("//h2[@class='title text-center']");
-
+    private final By scrollUpArrow_button = By.xpath("//i[@class='fa fa-angle-up']");
+    private final By slideHeading_div= By.xpath("(//h2)[2]");
```

Comment:
I believe we can have a more relative locator here.

Reply:
Done